src/test/java/yuquiz/user/service/UserServiceTest.java

```diff
@@ -46,4 +72,69 @@ void signUpTest() {
         // then
         verify(userRepository, times(1)).save(any(User.class));
     }
+
+    @Test
+    @DisplayName("회원 정보 조회 테스트")
+    void getUserInfoTest() {
+        // given
+        given(userRepository.findById(userId)).willReturn(Optional.ofNullable(user));
+
+        // when
+        UserDetailsRes userDetailsRes = userService.getUserInfo(userId);
+
+        // then
+        verify(userRepository, times(1)).findById(userId);
+        assertNotNull(userDetailsRes);
+        assertEquals(userDetailsRes.username(), user.getUsername());
+        assertEquals(userDetailsRes.email(), user.getEmail());
+        assertEquals(userDetailsRes.nickname(), user.getNickname());
```

Comment:
assert구문에서 매개변수의 기본적인 순서는 (expected, actual) 이라고 합니다.
즉, (나와야하는 값, 실제 테스트를 통해 얻은 값) 이 되어야한다고 합니다.
로직은 문제가 없지만 다른 부분들도 이렇게 되어있기도 하고 더 좋은 코드를 위해 남깁니다.

```suggestion
        assertEquals(user.getUsername(), userDetailsRes.username());
        assertEquals(user.getEmail(), userDetailsRes.email());
        assertEquals(user.getNickname(), userDetailsRes.nickname());
```

Reply:
그러네요 이부분만 이렇게 되어있던데 찾아주셔서 감사해요!
바로 수정했습니다